abstar/core/abstar.py

```diff
@@ -774,7 +786,7 @@ def _run_jobs_singlethreaded(files, output_dir, log_dir, file_format, args):
 
 
 def _run_jobs_via_multiprocessing(files, output_dir, log_dir, file_format, args):
-    p = Pool(processes=args.num_cores, maxtasksperchild=50)
```

Comment:
As explained in the PR description, passing `maxtasksperchild` parameter on the Linux system where the default startmethod for `multiprocessing` is `fork` is unstable, and has caused abstar runs to hang on numerous occasions. Removing this seems to have fixed the issue so far.

This conversation for example highlights the issue: https://github.com/python/cpython/issues/93580